originals/t.yaml

```
@@ -19,6 +19,20 @@
     genres:
     - Rhythm
 
+- name: Tails Adventure
+  names:
+  - テイルスアドベンチャー
+  external:
+    website: 'https://www.mobygames.com/game/35668/tails-adventure/'
+    wikipedia: Tails Adventure
+  platforms:
+  - 3DS
```

```suggestion
  - 3DS
  - Game Gear
```
will need to update schema also